README.md

```diff
@@ -6,7 +6,7 @@
 
 * [Narek Hovhannisyan](https://www.facebook.com/narek.hovh) - [OCaml for the sceptics](http://renderforest-ocaml.surge.sh/)
 * [Khachatur Virabyan](https://twitter.com/kh_mugh) - [Reasonable React](http://renderforest-reason.surge.sh/)
-* [Nairi Harutyunyan](https://www.facebook.com/harutyunyan.nairi) - [Redux & RxJS](https://slides.com/nairi/deck#/)
+* [Nairi Harutyunyan](https://www.facebook.com/harutyunyan.nairi) - [Redux & RxJS](https://slides.com/nairihar/deck#/)
```

Comment:
I'm getting error `The page you were looking for doesn't exist.`, it's ok?

Reply:
Oops, I have updated file, now is ok.